fs/communitarian-discourse-plugin
# [#174355206] Move new resolutions to top

https://www.pivotaltracker.com/story/show/174355206

## How to test
Create some resolution, or select existing one.
Visit http://localhost:3000/sidekiq/scheduled and trigger it's `Jobs::ReopenResolution` job (`Jobs::ClosePoll` is optional here).
You should see resolutions ordered by `created_at` time. Fresh ones should go first, the freshest should be large and you should be able to vote on it.
